src/InteractiveWindow/Editor/InteractiveWindow.csproj

```diff
@@ -98,12 +103,9 @@
     <Compile Include="InteractiveWindow.SystemClipboard.cs" />
     <Compile Include="ProjectionBufferExtensions.cs" />
     <Compile Include="InteractiveWindow.PendingSubmission.cs" />
-    <Compile Include="InteractiveWindow.ReplSpanKind.cs" />
     <Compile Include="InteractiveWindow.EditResolver.cs" />
     <Compile Include="InteractiveWindow.State.cs" />
     <Compile Include="InteractiveWindow.UIThreadOnly.cs" />
-    <Compile Include="PredefinedInteractiveContentTypes.cs" />
```

Comment:
Sorry, I didn't realize you meant these ones when we talked.  We shouldn't move these (they shipped in VS 2015 RTM).  If we need to duplicate the string literal, then we should do that.